docker/Dockerfile

```diff
@@ -63,7 +63,7 @@ RUN TORCH_VER=$(curl --silent --location https://pypi.org/pypi/torch/json | pyth
     else \
         pip install --no-cache-dir torch==$TORCH_VER+cpu torchvision==$TORCH_VISION_VER+cpu -f https://download.pytorch.org/whl/torch_stable.html; \
     fi
-RUN pip install --no-cache-dir captum torchtext torchserve torch-model-archiver
+RUN pip install -U setuptools && pip install --no-cache-dir captum torchtext torchserve torch-model-archiver
```

Comment:
Nit: is setuptools the only utility required to support custom pip requirements in docker?